toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -518,7 +518,9 @@ func generateContainerManifests(installChroot *safechroot.Chroot) {
 
 	os.MkdirAll(rpmManifestDir, os.ModePerm)
 
+	// 15 Dec 2022: Please contact Qualys before changing the following rpm query.
 	shell.ExecuteAndLogToFile(manifest1Path, "rpm", "--dbpath", rpmDir, "-qa")
+	// 15 Dec 2022: Please contact Qualys, AquaSec (trivy) and other supported scanning vendors before changing the following rpm query.
```

Comment:
Nit-pick: we don't add dates to our comment, let's stick with the convention. Also, git saves this information for us anyway.
```suggestion
	// Please contact Qualys before changing the following rpm query.
	shell.ExecuteAndLogToFile(manifest1Path, "rpm", "--dbpath", rpmDir, "-qa")
	// Please contact Qualys, AquaSec (trivy) and other supported scanning vendors before changing the following rpm query.
```